Workbooks/AKS/Workload Details AKS/Workload Details AKS.workbook

```diff
@@ -382,7 +434,7 @@
             },
             "timeContextFromParameter": "timeRange",
             "queryType": 0,
-            "resourceType": "microsoft.containerservice/managedclusters"
+            "resourceType": "microsoft.operationalinsights/workspaces"
```

Comment:
any reason for making workload workbooks are workspace centric instead of resource centric?

Reply:
fixed